.github/workflows/e2e_tests.yml

```
@@ -41,10 +45,13 @@ jobs:
           result =
             if event_name == "deployment_status" && event_state == "success"
               # Production deployment
-              if (environment_val.include?("Production") && target_url.include?("liam-app-git-main"))
+              if environment_val.include?("Production") && target_url.include?("liam-app-git-main")
                 "should_run=true"
               # Preview deployment
-              elsif environment_val.include?("Preview") && !target_url.include?("liam-erd-sample") && !target_url.include?("liam-docs") && !target_url.include?("liam-storybook")
+              elsif environment_val.include?("Preview") && (
+                target_url.include?("liam-app") || 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)